metpy/calc/tests/test_indices.py

```diff
@@ -21,7 +21,8 @@ def test_precipitable_water():
     """Test precipitable water with observed sounding."""
     with UseSampleData():
         data = get_upper_air_data(datetime(2016, 5, 22, 0), 'DDC', source='wyoming')
-    pw = precipitable_water(data.variables['dewpoint'][:], data.variables['pressure'][:])
+    pw = precipitable_water(data.variables['dewpoint'][:], data.variables['pressure'][:],
+                            top=400 * units.hPa)
```

Comment:
I'm not understanding at all how this change + the other few lines is enough to keep things working both *before* and *after*. Makes me think we're missing some test cases...

Reply:
Much better now.